src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/PrisonConfigController.kt

```diff
@@ -93,11 +130,114 @@ class PrisonConfigController(
       ),
     ],
   )
-  fun getPrison(
+  fun getPrisons(): List<PrisonDto> {
+    return prisonConfigService.getPrisons()
+  }
+
+  @PreAuthorize("hasRole('VISIT_SCHEDULER')")
```

Comment:
do we update the role to an admin role now? Could leave it i till later though.

Reply:
it's another task